lib/banking_calendar/data/bsp.yml

```diff
@@ -109,5 +109,26 @@ bank_holidays:
   - 2024-12-08
   - 2024-12-24
   - 2024-12-25
+  - 2024-12-26
   - 2024-12-30
   - 2024-12-31
+  - 2025-01-01
+  - 2025-01-02
```

Comment:
I don't think Jan 2 is a holiday. Can you double check?

Reply:
Hmm, we make it a holiday so it won't generate a payout right?